py/py2/ocGraph.py

```diff
@@ -70,14 +74,26 @@ def textwidth(text, fontsize=14):
         return len(text) * fontsize
     surface = cairo.SVGSurface('data/undefined.svg', 600, 600)
     cr = cairo.Context(surface)
-    cr.select_font_face('sans-serif', cairo.FONT_SLANT_NORMAL, cairo.FONT_WEIGHT_BOLD)
+    cr.select_font_face(
+        'sans-serif', cairo.FONT_SLANT_NORMAL, cairo.FONT_WEIGHT_BOLD
+    )
     cr.set_font_size(fontsize)
-    x_bearing, y_bearing, width, height, x_advance, y_advance = cr.text_extents(text)
+    x_bearing, y_bearing, width, height, x_advance, y_advance = cr.text_extents(
+        text
+    )
     return width
 
 
 # Graph generation based on Teresa Schmidt's R script (2016)
-def generate(model_name, varlist, hide_iv, hide_dv, dv_name, full_var_names, all_higher_order):
+def generate(
+    model_name,
+    varlist,
+    hide_iv,
+    hide_dv,
+    dv_name,
+    full_var_names,
+    all_higher_order,
+):
```

Comment:
same here

Reply:
I'd agree it takes some getting used to, though it does make diffs clearer due to changes being on separate lines.